students/Cracolici_Jon/lesson02/JonCracolici_GridPrint.py

```diff
@@ -0,0 +1,68 @@
+#Jon Cracolici
+#Lesson 2 Gridprint Problem
+#UW Python Cert
+#
+#
+def gridprint0():
+#This is the super basic function with no arguments. It just builds the grid seen at the beginning of the problem.
+    cellnum0 = 2  #number of cells in a row or column. Assumed square.
+    lnum = 4     #number of dashes between vertices
+    rnum = cellnum0*(lnum+1)+1  #count of total row numbers
+    i = range(rnum)
+    r1 = cellnum0*("+ "+lnum*"- ")+"+"
+    r2 = cellnum0*("| "+lnum*"  ")+"|"
+    for x in i:
+        if x%(lnum+1)==0:
+            print(r1)
+        else:
+            print(r2)
+
+gridprint0()
+
+def gridprint1(L):
```

Comment:
You'll want to make function names snake_case and variables lower case: grid_print_1(x): for example

Reply:
ok I will remember that for the future